[Bug] [spark-connector-v2-example] No class found

### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-seatunnel/issues?q=is%3Aissue+label%3A%22bug%22) and found no similar issues.


### What happened

Running spark-connector-v2-example in idea develop environment failed, the error as the following image:

![image](https://user-images.githubusercontent.com/51053924/179434729-4c41e84e-6b0e-4eb8-a90f-e18a9a4558a8.png)

I think it maybe the dependencies in pom has some problems, I try to add the 3.0.X  `janino` dependency in pom and fixed it.

In whole project I see it had been added the janino dependency in pom, but the version is `3.1.X`, maybe the `org/codehaus/janino/InternalCompilerException` not be contained in `3.1.X`

In my option, we should add the `janino` 3.0.X dependency in pom.

### SeaTunnel Version

dev

### SeaTunnel Config

```conf
none
```


### Running Command

```shell
none
```


### Error Exception

```log
Caused by: java.lang.ClassNotFoundException: org.codehaus.janino.InternalCompilerException
	at java.net.URLClassLoader.findClass(URLClassLoader.java:382)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:418)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:355)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:351)
	... 47 more
```


### Flink or Spark Version

_No response_

### Java or Scala Version

_No response_

### Screenshots

![image](https://user-images.githubusercontent.com/51053924/179435232-86392bd4-8d01-41bc-8ef0-cf8a05e50c1b.png)


### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

--- a/pom.xml
+++ b/pom.xml
@@ -763,11 +763,6 @@
                 <version>${codehaus.jackson.version}</version>
             </dependency>
             <dependency>
-                <groupId>org.codehaus.janino</groupId>
-                <artifactId>janino</artifactId>
-                <version>${janino.version}</version>
-            </dependency>
-            <dependency>
                 <groupId>com.sun.jersey</groupId>
                 <artifactId>jersey-json</artifactId>
                 <version>${jersey.version}</version>

--- a/seatunnel-connectors-v2/connector-hive/pom.xml
+++ b/seatunnel-connectors-v2/connector-hive/pom.xml
@@ -87,6 +87,12 @@
         </dependency>
 
         <dependency>
+            <groupId>org.codehaus.janino</groupId>
+            <artifactId>janino</artifactId>
+            <version>${janino.version}</version>
+        </dependency>
+
+        <dependency>
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>
         </dependency>

--- a/seatunnel-examples/seatunnel-spark-connector-v2-example/pom.xml
+++ b/seatunnel-examples/seatunnel-spark-connector-v2-example/pom.xml
@@ -86,13 +86,6 @@
             <artifactId>lz4</artifactId>
             <version>1.3.0</version>
         </dependency>
-
-        <dependency>
-            <groupId>org.codehaus.janino</groupId>
-            <artifactId>janino</artifactId>
-            <version>3.0.16</version>
-        </dependency>
-
     </dependencies>
 
 </project>

--- a/seatunnel-examples/seatunnel-spark-connector-v2-example/pom.xml
+++ b/seatunnel-examples/seatunnel-spark-connector-v2-example/pom.xml
@@ -80,10 +80,17 @@
             <version>${spark.version}</version>
             <scope>${spark.scope}</scope>
         </dependency>
+
         <dependency>
             <groupId>net.jpountz.lz4</groupId>
             <artifactId>lz4</artifactId>
             <version>1.3.0</version>
+        </dependency>
+
+        <dependency>
+            <groupId>org.codehaus.janino</groupId>
+            <artifactId>janino</artifactId>
+            <version>3.0.16</version>
         </dependency>
 
     </dependencies>

--- a/tools/dependencies/known-dependencies.txt
+++ b/tools/dependencies/known-dependencies.txt
@@ -319,6 +319,7 @@ jakarta.validation-api-2.0.2.jar
 jakarta.websocket-api-1.1.2.jar
 jamon-runtime-2.4.1.jar
 janino-3.1.6.jar
+janino-3.0.9.jar
 java-xmlbuilder-0.4.jar
 javassist-3.18.1-GA.jar
 javassist-3.20.0-GA.jar

